src/store/index.ts

```diff
@@ -0,0 +1,72 @@
+import { createStore, createLogger } from "vuex";
+import { Dapp, Account, DappLocalState } from "@/types";
+import api from "@/api/dapps";
+
+const debug = process.env.NODE_ENV !== "production";
+
+export default createStore({
+	state: {
+		hasAlgoSigner: false,
```

Comment:
instead of `hasAlgoSigner`, we should have an enum:
```
enum WalletKind {
  None = 0
  AlgoSigner = 1,
  MyAlgo,
  WalletConnect,
}
```
and in the Store, instead of `hasAlgoSigner`:
```
walletKind: WalletKind.None
```